apps/web-giddh/src/assets/css/table.scss

```
@@ -35,9 +36,9 @@
         }
     }
     thead > .register-table-th > th {
-        color: $black !important;
+        color: var(--color-nero) !important;
         border-bottom: none !important;
-        background: $light-orange;
+        background: var(--color-table-primary-giddh-table-thead-register-table-th-th-background);
```

Avoid redundant words in class names
```suggestion
        background: var(--color-table-primary-giddh-table-thead-register-table-th-background);
```